Azure/azure-cli
# [AppService] `az functionapp list`: Fix when resource group is not passed

**Related command**
`az functionapp list`

**Description**
Currently, when the `az functionapp list` command is run without a resource group, there are some pagination API calls made that fail with a 502 error. In this PR, we have a workaround for the failed pagination API calls.

This command is needed for Centauri when running the `az functionapp devops-pipeline create` command.

**Testing Guide**
Run `az functionapp list` without any resource group information. 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
